lib/blip_ds.dart

```diff
@@ -49,6 +49,7 @@ export 'src/themes/texts/styles/ds_headline_small_text_style.theme.dart'
 export 'src/themes/texts/utils/ds_font_families.theme.dart' show DSFontFamilies;
 export 'src/themes/texts/utils/ds_font_weights.theme.dart' show DSFontWeights;
 export 'src/utils/ds_animate.util.dart' show DSAnimate;
+export 'src/utils/ds_bubble.util.dart' show DSBubbleUtils;
```

Comment:
I think that isn't necessary to export this widget because is a util for DS ony

what do you think?

Reply:
Done.